docs/guides/developer/docs/develop/development-tips.md

```diff
@@ -13,6 +13,8 @@ Every development environment has its quirks, so here are a few which have been
 * If your IDE stubbornly refuses to acknowledge that a class exists, even when you're sure it's there, try closing your
   IDE, then running `git clean -fdx`, then building.  This will ensure everything in your clone is up to date.  Also
   ensure you find your project workspace to make sure your IDE isn't keeping a cache of things.
+* Check if you have selected the correct java version in your IDE. Opencast requires Java 11, but your IDE might have
+  selected a different version.
```

Comment:
Java 17 should work by now.

Reply:
You are right, normally I use JDK 17 on my computer without any issues.